packages/core/src/node-utils/paths.ts

```diff
@@ -35,6 +37,7 @@ const Paths = (cwd: string, prefix: string): Paths => ({
   customType: (id: string) => ({
     value: () => path.join(Paths(cwd, prefix).value()),
     model: () => path.join(Paths(cwd, prefix).value(), id, "index.json"),
+    types: () => path.join(Paths(cwd, prefix).value(), id, "types.ts"),
```

Comment:
isn't the standard to call it `index.d.ts` ?

Reply:
As far as I know, `*.d.ts` files are used to provide type information about their eponymous JavaScript files (e.g. `createFoo.js` & `createFoo.d.ts`)

In this case, `types.ts` doesn't represent the type of the `index.js` component (or the exports from that file). It represents the types of the Slice object (the `index.js` component eventually uses that). With that in mind we went for `types.ts` as thought more appropriate.

Reply:
ok